openapi/components/schemas/CustomerAuthentication/CustomerJWT.yaml

```
@@ -16,7 +16,7 @@ properties:
     type: string
     readOnly: true
   invalidate:
-    description: Specifies whether to invalidate a token after an exchange is performed.
+    description: Specifies if to invalidate a token after an exchange is performed.
```

This one does not sound quite right. 
```suggestion
    description: Specifies whether to invalidate a token after an exchange is performed.
```

---

```suggestion
    description: Specifies if a token is invalidated after an exchange is performed.
```

---

Thanks for spotting these @Boydbueno 🙏